### ISSUE TITLE
Issue with MN Label EOL

### ISSUE THREAD

**Issue by heathhayward** [labordynamicsinstitute/qwi_schemas#93]
https://lehd.ces.census.gov/data/schema/V4.2.0/label_geography_mn.csv has an extra empty line at the end that's breaking the QWI Explorer loading scripts. I'll remove the extra line and go through the process of committing the change to the Redmine and pushing it to DEV for review.  I'll let you know if I run into any issues.

**Comment by heathhayward** [comment]
When making edits to the schema, should I replace "label_geography_mn.csv" in ../external/github/V4.2.0 or ../post-processing/web/V4.2.0 or both? I'll then rerun "./prepare-web.sh start" after making the fix.

**Comment by heathhayward** [comment]
[label_geography_nm.zip](https://github.com/labordynamicsinstitute/qwi_schemas/files/2069131/label_geography_nm.zip)

**Comment by heathhayward** [comment]
bah, added the wrong file (NM instead of MN).  Let me try again.

**Comment by heathhayward** [comment]
Is there any cleanup that needs to be done for the NM label that I erroneously committed?  It doesn't look like there were any differences between the files...

**Comment by larsvilhuber** [comment]
I can (and have to) take it from here.


--
Lars Vilhuber, Economist
Cornell University, Executive Director, Labor Dynamics Institute
  and ILR School - Department of Economics
American Economic Association - Data Editor
Journal of Privacy and Confidentiality - Managing Editor

e: lars.vilhuber@cornell.edu
p: +1.607-330-5743
v: https://cornell.zoom.us/my/larsvilhuber
w: http://lars.vilhuber.com/ <http://lars.vilhuber.com/>

Assistant: ldi@cornell.edu | +1.607-255-2744
________________________________

**Comment by larsvilhuber** [comment]
The files you uploaded do not have the extra line (good) but they also lack the last end-of-line character (bad). I'm fixing that.

**Comment by heathhayward** [comment]
OK, thanks for your help with this.  Let me know when I can svn this final corrected version.
